app/src/main/AndroidManifest.xml

```diff
@@ -36,7 +36,8 @@
             android:name=".circle_of_trust.Trustees"
             android:label="@string/title_activity_trustees"
             android:parentActivityName=".MainActivity"
-            android:windowSoftInputMode="stateHidden">
+            android:windowSoftInputMode="adjustPan"
```

Comment:
Is there any reason you remove the statehidden here rather than putting  android:windowSoftInputMode="adjustPan|stateAlwaysHidden"